examples/complete/main.tf

```diff
@@ -373,7 +413,7 @@ module "eks" {
   # "native" EKS Marketplace Add-Ons
   #---------------------------------------------------------------
 
-  cluster_addons = var.cluster_addons
+  cluster_addons = local.cluster_addons
```

Comment:
does the cluster_addons var need to be removed from variables.tf?

Reply:
This logic is meant to merge IRSA role for vpc-cni cluster_addons values since you can't do real logic in tfvar files.

Reply:
I'm an idiot